frame/nfts/src/benchmarking.rs

```diff
@@ -483,16 +483,17 @@ benchmarks_instance_pallet! {
 		let price = ItemPrice::<T, I>::from(100u32);
 		let price_direction = PriceDirection::Receive;
 		let price_with_direction = PriceWithDirection { amount: price, direction: price_direction };
-		let duration = T::BlockNumber::max_value();
-	}: _(SystemOrigin::Signed(caller.clone()), collection, item1, collection, Some(item2), Some(price_with_direction.clone()), Some(duration))
+		let duration = T::MaxDeadlineDuration::get();
+	}: _(SystemOrigin::Signed(caller.clone()), collection, item1, collection, Some(item2), Some(price_with_direction.clone()), duration)
 	verify {
+		let expect_deadline = duration + frame_system::Pallet::<T>::block_number();
```

Comment:
Maybe use a `saturating_add` here and below in case someone uses `u64::MAX` as duration.

Reply:
done